frontend/widgets/OBSBasic.cpp

```diff
@@ -1372,8 +1373,10 @@ void OBSBasic::OnFirstLoad()
 
 OBSBasic::~OBSBasic()
 {
-	if (!handledShutdown) {
-		applicationShutdown();
+	blog(LOG_INFO, "[OBSBasic] Destructor");
+	if (!isClosing()) {
+		blog(LOG_INFO, "[OBSBasic] Destructor calling closeWindow");
+		closeWindow();
```

Comment:
What's the value for end users of these messages (as they are "INFO" level)?

Reply:
These are purely for test purposes and in a separate commit for removal later.